_gsocproposals/2020/proposal_ATLASMLcompression.md

```diff
@@ -0,0 +1,62 @@
+
+It will also be interesting to treat this study as a proof-of-principle for future data compression techniques for the ATLAS experiment. For the planned experimental upgrades in 2026, the techniques used in this report may help solving the problem of needing much more storage space than in the past due to the increase of the size of the dataset. 
+
+## Project Milestones 
+
+   * Run and benchmark an existing autoencoder network to compress ATLAS data.
+      * Benchmarking includes and is not necessarily limited to:
+         * The costs of training and decompressing the data in the context of a resource-constrained system such as the ATLAS trigger;
+         * The fidelity of decompressed data with respect to original inputs, in terms of impact on the quality of the physics analysis with de-compressed objects; 
+         * The dependence of the quality of the output on the similarity between training sample and test sample.
+   * Understand whether the network can be used in combination with other compression algorithms already used at the LHC, and whether this can increase the compression factor overall.
+   * The performance of combinations of compression algorithms will be evaluated using the same figures of merit as above. 
+   * If time allows, study the performance of the network as an anomaly detection algorithm, injecting simulated signal data into background data. 
+
+## Expected results
+
+The first milestone will lead to a set of plots made in MatPlotLib/ROOT that demonstrate the performance of the existing network on a different sample of ATLAS data with respect to what was used before for prototyping the network. 
+The second milestone will lead to the same plots as above, but in the case when the network is fed variables to compress that have already been compressed with another algorithm beforehand. Particular attention will be paid to CPU cost and speed of the network.  Ideally, these plots will point to a configuration that is viable for use on already-compressed data. 
+The third milestone will lead to a set of plots of the reconstruction error of a signal that is very different from the high-rate backgrounds that are being compressed, and increase the understanding of how the network treats anomalous signals when compressing it. 
+
+## Requirements
+
+Required: Good knowledge of UNIX, Python, MatPlotLib or ROOT, ability to install software and run a Jupyter notebook locally
+
+Desirable: Knowledge of ROOT and PyTorch, batch submission algorithms
+
+## Evaluation tasks
+
+Jupyter and pyTorch exercise. 
+
+Running the existing NN. 
+
+Making plots in MatPlotLib/ROOT. 
+
+## Links
+  * [1] [Zhe Zhang and Brian Bockelman. Exploring compression techniques for ROOT IO. CoRR (2017)](https://arxiv.org/abs/1704.06976)
+  * [ROOT](https://root.cern/)
+  * [Jupyter](http://jupyter.org)
+  * [PyTorch](http://pytorch.org)
+
+## Mentors
+  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch)
+  * Two more TBC
```

Comment:
We need at least one more mentor before merging. Please make sure to update also mentors.md after adding them.

Reply:
Added two mentors, here and on the mentors.md.